src/components/common/Cart/index.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import { useContext } from 'react'
+import { CartContext } from '@/contexts/CartProvider'
+import { mc } from '@/utils/functions/common'
+
+export default function Cart() {
+  const { isCartOpen, setIsCartOpen } = useContext(CartContext)
+
+  const closeCart = () => setIsCartOpen(false)
```

Comment:
the context should export openCart and closeCart function instead of the setIsCartOpen function